todo_app/src/Data/trello_items.py

```diff
@@ -0,0 +1,92 @@
+from tokenize import String
+from typing import Iterator
+from todo_app.src.Data.item import Item
+
+import requests
+import os
+
+
+class TrelloItems:
+    def __init__(self):
+        self._boardId = os.getenv('TRELLO_BOARDID')
+        self._key = os.getenv('TRELLO_KEY')
+        self._token = os.getenv('TRELLO_TOKEN')
+        self._default_list = 'To Do'
+
+    def change_status(self, title, status):
+
+        cards = self.__get_board_cards(self._boardId)
+        cardId = None
+        for card in cards:
+            if card['name'] == title:
+                cardId = card['id']
+        if cardId == None:
+            return
+
+        lists = self.__get_board_lists(self._boardId)
+        listId = None
+        for list in lists:
+            if list['name'] == status:
+                listId = list['id']
+        if listId == None:
+            return
+        
+        self.__set_card_list(cardId, listId)
+
+
+    def get_statuses(self) -> Iterator[str]:
+        lists = self.__get_board_lists(self._boardId)
+        for list in lists:
+            yield list['name']
```

Comment:
I think your code would be a bit simpler just to return a list instead of an iterator. You only call it in one place, and in that place you immediately convert it into a list.

So why not just `return [ list['name'] for list in lists ]` and skip the conversion in app.py